### ISSUE TITLE
Doesn't work as a reliable TimePicker only

### ISSUE THREAD

**Issue by jiggneshhgohel** [Eonasdan/bootstrap-datetimepicker#1685]
My requirement is to show a start of day i.e. 12:00 AM as the default time selected in the time-picker text field and just display the time-picker.

I was able to use just the time-picker using the `format` option but my core requirement of setting the default time to start of day was not fulfilled. Below given are the versions of jQuery

**bootstrap-datetimepicker v4.17.37**
**jQuery v1.12.0**
**Bootstrap v3.3.6**
**Browsers:** 
- **Mozilla Firefox 47.0 (on Ubuntu 14.04 LTS)**
- **Google Chrome Version 49.0.2623.87 (64-bit) (on Ubuntu 14.04 LTS)**

**JS**

```
  enableTimePicker = function() {
    var timeInputFieldObjects = $("input.time-picker");

    if (timeInputFieldObjects.length > 0) {
      timeInputFieldObjects.each(function() {
        $(this).datetimepicker({
          format: 'h:mm A',
          useCurrent: false,
          //defaultDate: moment().startOf('day')
          minDate: moment().startOf('day')
        });
      })
    }
  };
```

**Options used and steps performed and the outcome**:

**passing no options except `format` OR passing `format` and `useCurrent: false` OR passing `format` and `useCurrent: false` and `minDate: moment().startOf('day')`**

1) when the datetimepicker is applied the target text-field is empty
2) when focusing into the datetimepicker using the mouse and selecting a time, say 12:00 AM, 12:00 AM is shown in the text-field
3) when focusing out using mouse and focusing into again into the datetimepicker using the mouse but not selecting any time, 2:30 PM is shown
4) when focusing out using mouse and focusing into again into the datetimepicker using the mouse but not selecting any time, 3:00 PM is shown
5) when focusing out using mouse and focusing into again into the datetimepicker using the mouse but not selecting any time, 2:30 PM is shown

and subsequent focusing in and out keeps on switching the time between 2:30 PM and 3:00 PM. Don't know why 2:30 PM is displayed. I am in IST (UTC +05:30) timezone and at time of testing it the time was around 1:00 PM.

**passing `format` and `useCurrent: false` and `defaultDate: moment().startOf('day')`**

1) when the datetimepicker is applied the target text-field shows 2:30 PM as the default time. I was expecting it to be set to 12:00 AM
2) when focusing into the datetimepicker using the mouse, 2:30 PM gets replaced with 3:00 PM is shown in the text-field 
3) when focusing out using mouse and focusing into again into the datetimepicker using the mouse but not selecting any time, 2:30 PM is shown

and subsequent focusing in and out keeps on switching the time between 2:30 PM and 3:00 PM.

**Comment by jiggneshhgohel** [comment]
If anybody ends-up having the same requirement as I have and facing problem using this component as time-picker only I have successfully implemented my requirement using [this alternative](https://jdewit.github.io/bootstrap-timepicker/) component. And it works smoothly without any issues.

Hope it benefits someone urgently looking for solution until the issues in this library I have reported gets fixed.

Thanks.
